modules/social_features/social_event/asyncapi.yml

```diff
@@ -0,0 +1,139 @@
+                          type: string
+                          description: The display name of the author.
+                        href:
+                          type: object
+                          properties:
+                            canonical:
+                              type: string
+                              format: uri
+                              description: Canonical URL of the author.
+                    allDay:
+                      type: boolean
+                      description: Indicates if the event lasts all day.
+                    start:
+                      type: string
+                      format: date-time
+                      description: The start time of the event.
+                    end:
+                      type: string
+                      format: date-time
+                      description: The end time of the event.
+                    timezone:
+                      type: string
+                      description: The timezone of the event.
+                    address:
+                      type: object
+                      properties:
+                        label:
+                          type: string
+                          description: The label of the event location.
+                        countryCode:
+                          type: string
+                          description: The country code of the event location.
+                        administrativeArea:
+                          type: string
+                          description: The administrative area of the event location.
+                        locality:
+                          type: string
+                          description: The locality of the event location.
+                        dependentLocality:
+                          type: string
+                          description: The dependent locality of the event location.
+                        postalCode:
+                          type: string
+                          description: The postal code of the event location.
+                        sortingCode:
+                          type: string
+                          description: The sorting code of the event location.
+                        addressLine1:
+                          type: string
+                          description: The first address line of the event location.
+                        addressLine2:
+                          type: string
+                          description: The second address line of the event location.
+                    enrollment:
+                      type: object
+                      properties:
+                        enabled:
+                          type: boolean
+                          description: Indicates if enrollment is enabled.
+                        method:
+                          type: string
+                          description: The method of enrollment.
```

Comment:
```suggestion
                        method:
                          type: string
                          description: The method of enrollment.
                          enum:
                          - open
                          - invite
                          - request
```

Reply:
I also added a suggestion for this one based on what you did above with `enum` @tregismoreira

Reply:
pushed :)